source/lex.tex

```diff
@@ -143,7 +143,7 @@
 token\iref{lex.token}. The resulting tokens are syntactically and
```

Comment:
ISO explicitly called out the footnote on line 60. Perhaps:

"Implementations behave as if these separate phases occur, although in practice different phases can be folded together."

Reply:
fixed